go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/covalent-indexer-go v1.0.6
 	github.com/ElrondNetwork/elastic-indexer-go v1.2.25
-	github.com/ElrondNetwork/elrond-go-core v1.1.16-0.20220414130405-e3cc29bc7711
+	github.com/ElrondNetwork/elrond-go-core v1.1.16-0.20220523150518-f1519c41d352
```

Comment:
does this commit contains changes from both implementations?

Reply:
Yep, I merged hbv2 into partal-mb-execution in elrond-go-core